src/main/flaskr/terminal/endpoint/terminal_endpoint.py

```diff
@@ -32,7 +26,10 @@ def start_command():
     Returns:
         text_response: [Int, String]
     """
+    from src.main.bluesky.Client import connectbs
     res = text_response
+    runbluesky()
+    connectbs()
```

Comment:
Up for debate if we want to keep bluesky in this project...

Reply:
i have deleted the bluesky stuff